wordfreq/_chinese_mapping.py

```diff
@@ -0,0 +1,3275 @@
+SIMPLIFIED_MAP = {13423: '㑔',
```

Comment:
How long does it take to load this file?  If significant, I wonder if it might actually load faster as compressed msgpack.  (It would almost certainly save space, anyway.)